install_config/topics/using_gcp_load_balancer-proc.adoc

```diff
@@ -0,0 +1,113 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_gce.adoc
+////
+
+= Using the GCP external load balancer as a service
+
+{product-title} can leverage the GCP load balancer capacity by exposing services
+externally using a `LoadBalancer` service. {product-title} creates the load
+balancer in GCP and creates the proper firewall rules.
+
+[NOTE]
+====
+Currently, port 10256 must be opened manually for service load balancers
+to work in GCP. See https://bugzilla.redhat.com/show_bug.cgi?id=1583833 for
+more information.
+====
+
+.Procedure
```

Comment:
Replace the .Procedure and 'create a loadbalancer' service sentence by:

The following snippet is an example of a `LoadBalancer` service: